src/seedu/addressbook/data/person/Person.java

```diff
@@ -9,12 +9,14 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class Person implements ReadOnlyPerson {
-
+	
+	private static int nextSequenceNumber = 1;
+	private int sequenceNumber;
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)